pom.xml

```diff
@@ -483,6 +483,14 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+        <repository>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+            <id>repo-cuba-platform-work</id>
+            <name>repo</name>
+            <url>https://repo.cuba-platform.com/content/groups/work</url>
+        </repository>
```

Comment:
What is the purpose of this?

Reply:
Repository for library to generate reports
https://github.com/cuba-platform/yarg